src/views/templates/details/tabs/details/components/Description.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import produce from 'immer';
+
+import { TemplateModel, V1Template } from '@kubevirt-ui/kubevirt-api/console';
+import { DescriptionModal } from '@kubevirt-utils/components/DescriptionModal/DescriptionModal';
+import { useModal } from '@kubevirt-utils/components/ModalProvider/ModalProvider';
+import { useKubevirtTranslation } from '@kubevirt-utils/hooks/useKubevirtTranslation';
+import { k8sUpdate } from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Button,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  Text,
+} from '@patternfly/react-core';
+import { PencilAltIcon } from '@patternfly/react-icons';
+
+type DescriptionProps = {
+  template: V1Template;
+  editable: boolean;
+};
+
+const Description: React.FC<DescriptionProps> = ({ template, editable }) => {
+  const { createModal } = useModal();
+  const { t } = useKubevirtTranslation();
+  const templateDescription = template?.metadata?.annotations?.description || t('None');
+
+  const updateDescription = (updatedDescription: string) => {
+    const updatedTemplate = produce<V1Template>(template, (templateDraft: V1Template) => {
+      if (!templateDraft.metadata.annotations) templateDraft.metadata.annotations = {};
+
+      if (updatedDescription) {
+        templateDraft.metadata.annotations.description = updatedDescription;
+      } else {
+        delete templateDraft.metadata.annotations.description;
+      }
+      return templateDraft;
+    });
+
+    return k8sUpdate({
+      model: TemplateModel,
+      data: updatedTemplate,
+      ns: updatedTemplate?.metadata?.namespace,
+      name: updatedTemplate?.metadata?.name,
+    });
+  };
+
+  const onEditClick = () =>
+    createModal(({ isOpen, onClose }) => (
+      <DescriptionModal
+        obj={template}
+        isOpen={isOpen}
+        onClose={onClose}
+        onSubmit={updateDescription}
+      />
+    ));
+
+  return (
+    <DescriptionListGroup>
+      <DescriptionListTerm>{t('Description')}</DescriptionListTerm>
+      <DescriptionListDescription>
+        {editable ? (
+          <Button type="button" isInline onClick={onEditClick} variant="link">
+            {templateDescription}
```

Comment:
description should not be clickable
a. it may be a long text
b. it should be black

Reply:
Thanks, I will update the PR